caniuse.md

```diff
@@ -4,67 +4,138 @@ Track the implementation status of the AOM in various browsers.
 
 ## How to enable AOM
 
-*Chrome*:
+**Chrome**:
+*For `AccessibleNode`/`ComputedAccessibleNode`-related features:*
 ```--enable-blink-features=AccessibilityObjectModel```
 
-*Safari Technology Preview*:
+*For web platform related features:*
+Browse to `chrome://flags`, enable `enable-experimental-web-platform-features`.
+
+**Safari Technology Preview**:
 ```Develop > Experimental Features > Accessibility Object Model```
 
-*Firefox*:
+**Firefox**:
 ```about:config accessibility.AOM.enabled = true```
 
 ## Summary
 
-Last updated: May 8, 2018
+Last updated: July 9, 2019
 
 | | Chrome | Safari | Firefox |
 | --- | --- | --- | --- |
-| Phase 1: Reflect ARIA attributes on DOM nodes. | **Yes**, out-of-date syntax | **No** | **No** |
-| Phase 2: Listen for input events from AT. | **Yes**, out-of-date syntax, 6 events | **Yes**, 8 events | **No** |
+| Phase 1: Reflect ARIA attributes on DOM nodes. | **Yes**, behind experimental-web-platform-features flag | **Yes** | **No** |
+| Phase 1: Reflect element references for IDREF attributes | **No** | **No** | **No** |
+| Phase 1: Custom element semantics on `ElementInternals` | **No** | **No** | **No** |
+| Phase 2: Generated fallback events for AT actions | **No** | **No** | **No** |
+| Phase 2: New InputEvent types | **No** | **No** | **No** |
 | Phase 3: Build virtual accessible nodes. | **Yes**, out-of-date syntax | **No** | **No** |
 | Phase 4: Query computed accessibility tree. | **Yes**, out-of-date syntax | **No** | **Yes**, out-of-date syntax |
 
 ### Phase 1: Reflect ARIA attributes on DOM nodes.
 
-*Chrome*:
+*Available in Safari; behind `--experimental-web-platform-features` flag in Chrome*
 
+```js
+element.role = "button";
+element.ariaLabel = "Click Me";
 ```
-element.accessibleNode.role = "button";
-element.accessibleNode.label = "Click Me";
-element.accessibleNode.labeledBy = new AccessibleNodeList();
-element.accessibleNode.labeledBy.add(other.accessibleNode);
-...
-```
 
-### Phase 2: Listen for input events from AT.
+### Phase 1: Reflect element references for IDREF attributes
+
+*Not yet implemented*
```

Comment:
nothing review related just wanted to leave a :grimacing: face here.

Reply:
😬